### ISSUE TITLE
Toggle doesn't read its labels

### ISSUE THREAD

**Issue by betrue-final-final** [OfficeDev/office-ui-fabric-react#3362]
OS: Windows 10 Creators Update
Browser: Edge
Screen reader: Narrator

Are you willing to submit a PR to fix? (No)

Requested priority: (High) 

When using Narrator, tabbing to a toggle control only tells the state, and type, but not the name of the control instance. This makes them unusable for blind people.
